### ISSUE TITLE
PhantomJS unable to load the address

### ISSUE THREAD

**Issue by ariya** [ariya/phantomjs#10178]
_**[skau...@gmail.com](http://code.google.com/u/109088545579033650398/) commented:**_
> <b>Which version of PhantomJS are you using?</b>
1.2.0
> 
> <b>What steps will reproduce the problem?</b>
1. save screenshot.js as here: http://jsbin.com/udopaj/3/js
> 2. xxx:~$ phantomjs --load-plugins=yes --disk-cache=yes --ignore-ssl-errors=yes screenshot.js 'http://bookfi.org/' '1.png'
> 
> <b>What is the expected output? What do you see instead?</b>
Expected: silent success
> Real: 'Unable to load the adress!'
> 
> <b>Which operating system are you using?</b>
Debian 6.0.1
> 
> <b>Did you use binary PhantomJS or did you compile it from source?</b>
compiled
> 
> <b>Please provide any additional information below.</b>

> I saved bookfi mainpage source to my server and experimented with it. Problems caused by string:
> &lt;script type=&quot;text/javascript&quot; src=&quot;http://j.maxmind.com/app/geoip.js&quot;&gt;&lt;/script&gt;
> After commenting it all works fine.

**Disclaimer:**
This issue was migrated on 2013-03-15 from the project's former issue tracker on Google Code, [Issue #178](http://code.google.com/p/phantomjs/issues/detail?id=178).
:star2: &nbsp; **4** people had starred this issue at the time of migration.

**Comment by zackw** [comment]
This appears to be fixed in 2.0 -- at least, I get an error message from rasterize.js about a missing `geoip_country_code` variable but that does not prevent a screenshot being taken.

**Comment by dehlen** [comment]
I can not share this observation of yours. 
```
phantomjs --version : 2.0.0
```
OS: Mac OSX, Yosemite 10.10.5, phantomjs from binary download

Command:
```
./phantomjs --ignore-ssl-errors=true --ssl-protocol=any --web-security=true rasterize.js http://netflix.com test.png
```

Output:
```
Unable to load the address!
```

So maybe someone could have a look at it again ?

Best Regards,
David

**Comment by Vitallium** [comment]
Your example worked for me. I think the problem is in your ISP, network or even in your machine (SSL certificates).

Try to run it with `debug` flag enabled: `./phantomjs --debug=yes --ignore-ssl-errors=true --ssl-protocol=any --web-security=true rasterize.js http://netflix.com test.png`

**Comment by dehlen** [comment]
Sorry I just deleted the comment since this issue is closed and made a new one at #13830.
